build.gradle

```diff
@@ -52,6 +52,7 @@ dependencies {
     testImplementation('junit:junit:4.12')
     testImplementation('org.assertj:assertj-core:3.11.1')
     testImplementation('org.slf4j:slf4j-simple:1.7.28')
+    implementation( 'org.springframework.boot:spring-boot-starter-web:2.1.1.RELEASE' )
```

Comment:
- webin-cli-validator can't depend on the the whole spring-boot-starter-web as this will brings the whole Spring web service infrastructure and String command line programs (we do have these) will transform to web services in all projects that import this jar breaking them (we actually had exactly this bug already once this year).
- Do the same as in Webin-CLI i.e.
```
 implementation( 'org.springframework.boot:spring-boot-starter-web' ) {
        exclude group: 'org.springframework.boot', module: 'spring-boot-starter-tomcat'
    }
```